linux-deploy.sh

```diff
@@ -0,0 +1,21 @@
+# node should be installed
+
+if (which node)
+then 
+ echo "node is installed"
+else
+ echo "installing node"
+ apt install node
```

Comment:
This is good. If you want a more up to date version you can try this too:

https://github.com/nodesource/distributions#deb

Reply:
you'd put this in your script inside the "installing node" else statement:

```
curl -SLO https://deb.nodesource.com/nsolid_setup_deb.sh
chmod 500 nsolid_setup_deb.sh
./nsolid_setup_deb.sh 21
apt-get install nodejs -y
```

Reply:
You know what I seen that, and thought it to be too verbose. For what I was trying to do, I'll change it when I get by my machine.